play/PlayerRules/CharacterCreation/02_Classes.txt

```diff
@@ -2,645 +2,696 @@ Core classes:
+	You can have a MAX of 5 Miss Chance Defense from this feat.
+	*(If dual wielding, use the larger of the two Weapon Reflexes).
+[THE FOLLOWNG PASSIVES ONLY APPLY TO PISTOLS]
+Choose two bonuses and one weakness:
+    - Bonuses: +10 Dam, +8m R3 Range Bracket, -1 Miss Chance, +1 rds in mag, +2 mags carried;
+    - Weaknesses: -10 Dam, -8m R3 Range Bracket, +1 Miss Chance, -1 rds in mag, -2 mags carried;
+Weapon Proficiencies: Pistols.
+Armor Proficiencies: Concealed, Recon, Light.
+Starting Wealth: $850 + 100*Luck + 1125
 Feats:
-----------------------
-Heightened awareness -
-Being aware comes with your level of training. You cannot be caught flat-footed
-, but opponents can still get surprise rounds against you.
-|
-|
-|
-V
-You are omni-aware to any attacks made against you or allies. Whenever an 
-enemy attempts to make an attack of opportunity on you or any allies within eye
--sight, you gain an attack of opportunity on them.
-----------------------
-Enhanced Gun-Kata -
-You are a disciple of the art of close-range pistol fighting. Applies to any 
-target within 15m. On a successful hit, you may continue rolling hits until 
-you fail to hit or run out of rounds in your magazine. You must make each 
-successive hit on a different target. THIS MEANS ONE HIT PER TARGET PER KATA.
-|
-|
-|
-V
-You are a master of pistols. Applies to an target within 35m. On a successful 
-hit, you may continue rolling hits until you fail to hit. if you run out of 
-rounds, you may reload mid-kata by passing a DC 75% DEX check. You may hit a 
-target multiple times, but there must be one shot between shots fired at a 
-single target. (Ask GM).
-----------------------
-
-"Assassin" - The galaxy is a dangerous place for those who would believe their 
-power or influence are limitless. The assassin lurks in the shadows, either as 
-a lone force of justice, or as part of a larger organization. They have 
-rightfully earned the title of Galaxy''s greatest Hand to Hand Fighters.
-Passives: No penalty for engaging in Grappling maneuvers. 
-Assassins wear a Shadow Cloak, which allows them to become a mere silhouette. 
-When in partial darkness or darker, the static DC increases by 10.
-NOTICE: SEE BASIC RULES FOR ADDITIONAL CLOAKING ATTRIBUTES
-Weapon Proficiencies: Pistol, Hand to Hand Expert
-Armour Proficiencies: Concealed, Recon, Light, Cloaking.
-Starting Wealth: $1700+100*Luck - 1300
+	----------------------
+	Heightened Alertness -
+	Being alert comes with your level of training. Gain 10 Reflex. You cannot be caught 
+	flat-footed (you cannot be penalized for being surprised), but opponents can still 
+	get surprise rounds against you.
+	----------------------
+	Protective Awareness - Requires a minimum Reflex of 24
+	Assiduously observant, you stay aware of any attacks made against you or allies. 
+	Whenever an enemy attempts to make an opportunity attack on you or any allies within 
+	eye-sight, you first gain an attack on them.
```

Comment:
Yeah, I need to mention that fact that their weapon range is a limitation.
And on further review, it'd probably be prudent to put in some sort of cap -- maybe from player level, a hard cap, or nanite cost

Reply:
Also, my previous logic also includes the fact that the gunslinger is stuck with pistols which don't scale well into the late game against armored targets (unless using lasers, which have their own limitations such as a lower average base damage).

Reply:
@bleehu

Reply:
I think we can improve this, but this is not a blocking change. We can ship 2.0 and then nitpick this one feat. That's why my review said approved. 
I would like to see this interrupt cost an action on the following turn or something. I kinda think that if the gunslinger can sometimes aim and fire without it costing an action, then players will ask or subconsciously think "then why does it cost me an action to aim and an action to fire normally? Is it just because it's a game?" I want to be careful about trying to nuke everywhere where that sentiment crops up, because we'll end up going around in circles again about the balance between fun-realism, but I think there's enough cognitive dissonance here to ask for some cost besides just a really high reflex.